<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpViewDialog</class>
 <widget class="QDialog" name="HelpViewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Help view</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>55</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonBackward">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="teresources.qrc">
          <normaloff>:/images/icons/back.png</normaloff>:/images/icons/back.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonHome">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="teresources.qrc">
          <normaloff>:/images/icons/gohome.png</normaloff>:/images/icons/gohome.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonForward">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="teresources.qrc">
          <normaloff>:/images/icons/forward.png</normaloff>:/images/icons/forward.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>54</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="textBrowserHelp"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushButtonHome</tabstop>
  <tabstop>pushButtonBackward</tabstop>
  <tabstop>pushButtonForward</tabstop>
  <tabstop>textBrowserHelp</tabstop>
 </tabstops>
 <resources>
  <include location="teresources.qrc"/>
 </resources>
 <connections/>
</ui>
